.github/workflows/unittests.yml

```diff
@@ -6,9 +6,7 @@ name: Unit tests
 
 on:
   push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
Is this change not going to run the unittests twice for every PR push? Once for the `push` event, once for the `pull_request` event?

Reply:
Prepare to be disappointed